### DISCUSSION TITLE
line-number-current-line not themed

### DISCUSSION THREAD

**Discussion by gtms** [arcticicestudio/nord#281] [Development]
The top screenshot on the readme page shows the current line highlighted with a different face (line 20, in white), suggesting that `line-number-current-line` is themed. However, grepping for this variable throughout `nord-theme.el` does not yield any result.

As a result, nord-theme-20190616.1757 does not highlight the current light number as advertised. Is this a versioning issue?

**Comment by arcticicestudio** [comment]
Hi @gtms :wave:, thanks for your contribution :+1:
The screenshot was taken on my local system that uses the development state of the theme so this might be a bit confusing. The theme key will be added with the next release version.
I'll let this issue open in order to track the status and notify you when the key has been implemented.

**Reply by svengreb** [reply]
@gtms Thank you again for your contribution and patience! 🙏🏼
It's been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance). Please keep in mind that maintainers must maintain (pun intended 😁) balance for their open source work and life to avoid burnouts so this gap in time has nothing to do with you or your contribution.

I recently published the first ["Northern Post — The state and roadmap of Nord" announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a "clean state". Read the [announcement about reaching the "clean" contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord's discussions for more details about the goal.

Due to the scope of this issue of not being "purely technical", or being related to the project's code at all, I'll convert it into a discussion for [Nord's central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
